apps/website/src/styles/global.css

```
@@ -1,128 +1,224 @@
-    --color-card: var(--card);
-    --color-foreground: var(--foreground);
-    --color-background: var(--background);
-    --radius-sm: calc(var(--radius) * 0.6);
-    --radius-md: calc(var(--radius) * 0.8);
-    --radius-lg: var(--radius);
-    --radius-xl: calc(var(--radius) * 1.4);
-    --radius-2xl: calc(var(--radius) * 1.8);
-    --radius-3xl: calc(var(--radius) * 2.2);
-    --radius-4xl: calc(var(--radius) * 2.6);
+  --font-sans: 'Figtree Variable', sans-serif;
+  --font-display: Georgia, 'Times New Roman', serif;
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)